### PR TITLE
all_logic_order

### PR DESCRIPTION
**Pull request by datToc004**
## Related Tickets:
- [#40340](https://edu-redmine.sun-asterisk.vn/issues/40340)
- [#40344](https://edu-redmine.sun-asterisk.vn/issues/40344)
- [#40346](https://edu-redmine.sun-asterisk.vn/issues/40346)
- [#40348](https://edu-redmine.sun-asterisk.vn/issues/40348)
- [#40350](https://edu-redmine.sun-asterisk.vn/issues/40350)
- [#40352](https://edu-redmine.sun-asterisk.vn/issues/40352)
- [#40355](https://edu-redmine.sun-asterisk.vn/issues/40355)
- [#40357](https://edu-redmine.sun-asterisk.vn/issues/40357)
- [#40360](https://edu-redmine.sun-asterisk.vn/issues/40360)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by datToc004** [comment]
ready

File: app/Http/Controllers/Manager/OrderController.php [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    {
+        $orders = Order::paginate(config('restaurant.paginate.order'))->where('status', 0);
+
+        return view('manager.order.index', compact('orders'));
+    }
+
+    public function acceptOrder($id)
+    {
+        $order = Order::findOrFail($id);
+        $order->status = 1;
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/OrderController.php:29]: status 1 là gì? bỏ vào config nha

File: app/Http/Controllers/Manager/OrderController.php [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    {
+        $order = Order::findOrFail($id);
+
+        return view('manager.order.bill', compact('order'));
+    }
+
+    public function postBill(Request $request, $id)
+    {
+        $order = Order::findOrFail($id);
+        $order->status = 3;
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/OrderController.php:60]: đây nữa

**Comment by datToc004** [comment]
ready

File: app/Http/Controllers/Manager/OrderController.php [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+use App\Models\Table;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Session;
+
+class OrderController extends Controller
+{
+    public function index()
+    {
+        $orders = Order::paginate(config('restaurant.paginate.order'))->where('status', 0);
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/OrderController.php:21]: status 0 1 v.v... bỏ config nha

File: app/Http/Controllers/Manager/OrderController.php [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+    {
+        $order = Order::findOrFail($id);
+        $order->delete();
+
+        return redirect()->route('orders.index');
+    }
+
+    public function getAcceptOrders()
+    {
+        $orders = Order::paginate(config('restaurant.paginate.order'))->where('status', 1);
```
Comment (manhtd-1083) [inline app/Http/Controllers/Manager/OrderController.php:45]: status 0 1 v.v... bỏ config nha

**Comment by datToc004** [comment]
ready

File: app/Http/Middleware/CheckRole.php [hunk 5]
```diff
@@ -21,7 +21,7 @@ public function handle(Request $request, Closure $next, ...$roles)
             return redirect('login');
         }
         $user = Auth::user();
-        if (Auth::user()->role_id == 3) {
+        if (Auth::user()->role_id == 1) {
```
Comment (manhtd-1083) [inline app/Http/Middleware/CheckRole.php:24]: lại số rồi >:( comment đi comment lại bn lần rồi nè, bỏ vào config nha

**Comment by datToc004** [comment]
ready

**Review by manhtd-1083 — APPROVED** [review]

**Pull request merged**